components/base-css/src/_utility-classes.scss

```diff
@@ -1833,6 +1833,9 @@
 .font-lineheight-5 {
   line-height: var(--font-lineheight-5) !important;
 }
+.font-lineheight-6 {
+  line-height: var(--font-lineheight-5) !important;
```

Comment:
I think we should only use `!important` in the design system when in super rare cases because it makes subtheming difficult but since this is following an established pattern I think it's fine for now.

Reply:
We want to minimize use of important but I bet this utility class rule gets overriden by rules outside our design system that have superior specificity in a site that uses the CAWeb WordPress theme.

We could classify types of rules that we want to protect and others we expect to be overriden:

- Maybe font related rules should expect to be overriden easily by surrounding site rules so shouldn't use important
- Larger layout rules like display wouldn't be expected to get overriden casually so should use important

Not suggesting changes to this PR but appreciate the discussion

Reply:
Yes, utility classes are using important for those reasons. People who use utility classes need to be sure that they are strong enough.  Everywhere else we definitely need to avoid it.